src/dom.js

```diff
@@ -116,7 +116,7 @@ export function waitForHeadPromise(doc) {
  */
 export function waitForBody(doc, callback) {
   waitForChild(doc.documentElement, () => (
-    isDocumentReady(doc) && !!doc.body
+    onDocumentReady(doc)
```

Comment:
Sorry, `waitForBody` just calls `onDocumentReady`. There's essentially aliases at this point:

```js
export function waitForBody(doc, callback) {
  return onDocumentReady(doc, callback);
}
```

Same for `waitForBodyPromise`.

Reply:
I think my most recent commit has that change, but tests still seem to be failing for some reason